templates/Datashades-OpsWorks-CKAN-Stack.cfn.yml.j2

```diff
@@ -494,50 +562,6 @@ Resources:
         Type: !Ref CookbookURLType
         # SshKey: !Sub "${CookbookSSHKey}\n"
         Url: !Ref CookbookURL
-      CustomJson:
```

Comment:
does opsworks need this when we are transitioning?

Reply:
Not since cookbook 7.0.4. It's all in Parameter Store now.